### PR TITLE
Added support for description, iin, and issuer attributes on the Card model.

### PR DESCRIPTION
**Pull request by andrewnelder**
* Modified Card model to include: `description`, `iin`, and `issuer` attributes.
* Added version bump to v2.8.1

### CODE REVIEW

File: src/main/java/com/stripe/model/Card.java [hunk 1]
```diff
@@ -34,6 +34,12 @@
 	String status;
 	String tokenizationMethod;
 
+	// Please note that these field are for internal use only and are not typically returned
+	// as part of standard API requests.
+	String description;
```
Comment (zackstripe) [inline src/main/java/com/stripe/model/Card.java]: Way outside of the scope of this PR but as we're following the getters / setter idiom we should declare these as private.

File: src/main/java/com/stripe/model/Card.java [hunk 2]
```diff
@@ -221,6 +227,18 @@ public void setTokenizationMethod(String tokenizationMethod) {
 		this.tokenizationMethod = tokenizationMethod;
 	}
 
+	public String getDescription() { return description; }
```
Comment (zackstripe) [inline src/main/java/com/stripe/model/Card.java]: Nit. Format is inconsistent with other getters/setters.

**Comment by zackstripe** [comment]
One nit, otherwise LGTM.

File: src/main/java/com/stripe/model/Card.java (hunk 1 above)
Comment (remistr) [inline src/main/java/com/stripe/model/Card.java]: @zackstripe I think it's due to the way we built the lib. See brandur's comment here: https://github.com/stripe/stripe-java/pull/309#discussion_r77237022

Comment (brandur) [inline src/main/java/com/stripe/model/Card.java]: +1. An explicit `private` would be better, but we these get `package-private` by default and at least won't leak out of the package.

File: src/main/java/com/stripe/model/Card.java (hunk 2 above)
Comment (brandur) [inline src/main/java/com/stripe/model/Card.java]: Committed a small patch in ad217ff to give these getter/setters a little more breathing room.

**Comment by brandur** [comment]
@andrewnelder Thanks!

BTW, for future reference, no need to bump the version everywhere in the project. We have scripts that'll do it do that for us and it's usually better to use them because they're not likely to miss anything by accident. I'll removed the version bumps here for now.

**Comment by brandur** [comment]
Released as 2.9.0.

**Pull request merged**